src/app/games/mancala/common/MancalaScoreHeurisic.ts

```diff
@@ -1,16 +1,25 @@
 import { MancalaState } from '../common/MancalaState';
-import { PlayerMetricHeuristic } from 'src/app/jscaip/AI/Minimax';
+import { HeuristicBounds, PlayerMetricHeuristicWithBounds } from 'src/app/jscaip/AI/Minimax';
 import { PlayerNumberTable } from 'src/app/jscaip/PlayerNumberTable';
 import { MancalaMove } from './MancalaMove';
 import { MancalaConfig } from './MancalaConfig';
 import { MGPOptional } from '@everyboard/lib';
 import { MancalaNode } from './MancalaRules';
+import { BoardValue } from 'src/app/jscaip/AI/BoardValue';
 
-export class MancalaScoreHeuristic extends PlayerMetricHeuristic<MancalaMove, MancalaState, MancalaConfig>
+export class MancalaScoreHeuristic extends PlayerMetricHeuristicWithBounds<MancalaMove, MancalaState, MancalaConfig>
 {
 
     public override getMetrics(node: MancalaNode, _config: MGPOptional<MancalaConfig>): PlayerNumberTable {
         return node.gameState.getScoresCopy().toTable();
     }
 
+    public override getBounds(config: MGPOptional<MancalaConfig>): HeuristicBounds<BoardValue> {
+        const maxScore: number = config.get().width * 2 * config.get().seedsByHouse;
+        return {
+            player0Max: BoardValue.ofSingle(maxScore, 0),
+            player1Max: BoardValue.ofSingle(0, maxScore),
```

Comment:
I'd wish to see asymetrical scoring there, for future probing

Reply:
2. why not "HeuristicWithBounds" ?

Reply:
1. what do you mean?
2. car c'est pas une heuristique avec bornes, c'est "les bornes de l'heuristique"